packages/web/src/components/FontAwesomeIcon.js

```diff
@@ -0,0 +1,16 @@
+// Module imports
+import { FontAwesomeIcon as OriginalFontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+
+
+
+
+export function FontAwesomeIcon(props) {
+	if (typeof window === 'undefined') {
+		return null
+	}
```

Comment:
This is the entire reason the local `FontAwesomeIcon` component exists. Next.js renders the app on both the client *and* the server, and the way we load FontAwesome doesn't jive well with server side rendering.